holoviews/plotting/plotly/dash.py

```diff
@@ -475,31 +475,39 @@ def update_figure(*args):
             graph_id = graph_ids[fig_ind]
             # plotly_stream_types
             for plotly_stream_type, uid_to_streams_for_type in uid_to_stream_ids.items():
-                panel_prop = plotly_stream_type.callback_property
-                if panel_prop == "selected_data":
-                    if graph_id + ".selectedData" in triggered_prop_ids:
-                        # Only update selectedData values that just changed.
-                        # This way we don't the the may have been cleared in the
-                        # store above
-                        stream_event_data = plotly_stream_type.get_event_data_from_property_update(
-                            selected_dicts[fig_ind], initial_fig_dicts[fig_ind]
-                        )
-                        for uid, event_data in stream_event_data.items():
-                            if uid in uid_to_streams_for_type:
-                                for stream_id in uid_to_streams_for_type[uid]:
-                                    store_data["streams"][stream_id] = event_data
-                elif panel_prop == "viewport":
-                    if graph_id + ".relayoutData" in triggered_prop_ids:
-                        stream_event_data = plotly_stream_type.get_event_data_from_property_update(
-                            relayout_dicts[fig_ind], initial_fig_dicts[fig_ind]
-                        )
-
-                        for uid, event_data in stream_event_data.items():
-                            if event_data["x_range"] is not None or event_data["y_range"] is not None:
+                for panel_prop in plotly_stream_type.callback_properties:
+                    if panel_prop == "selected_data":
+                        if graph_id + ".selectedData" in triggered_prop_ids:
+                            # Only update selectedData values that just changed.
+                            # This way we don't the the may have been cleared in the
```

Comment:
???

Reply:
Thanks! Yeah, that comment didn't make sense.  fixed in 2da9ef9